packages/@uppy/react/package.json

```
@@ -39,7 +39,7 @@
     "@uppy/file-input": "workspace:^",
     "@uppy/progress-bar": "workspace:^",
     "@uppy/status-bar": "workspace:^",
-    "react": "^18.0.0"
+    "react": "^18.0.0 || ^19.0.0-0"
```

Don't you mean this?

```suggestion
    "react": "^18.0.0 || ^19.0.0"
```